modules/mobileApp/registroMobile.log.ts

```diff
@@ -0,0 +1,4 @@
+import { Connections } from '../../connections';
+import { Logger } from '@andes/log';
+
+export const registroMobileLog = new Logger({ connection: Connections.logs, module: 'mobile', type: 'mobile-registroMobile', application: 'andes' });
```

Comment:
Si el modulo se llama **mobile** no hace falta repetir la palabra mobile en el tipo. 
```suggestion
export const registroMobileLog = new Logger({ connection: Connections.logs, module: 'mobile', type: 'registro', application: 'andes' });
```
```